src/segment-loader.js

```diff
@@ -504,6 +504,11 @@ export default class SegmentLoader extends videojs.EventTarget {
    *
    * @private
    */
+
+  disableTimeout() {
+    this.dontTimeout = true;
```

Comment:
We have been trying to avoid adding more and more state to the segment loader (even though we haven't been doing a great job at that!) Is there another way to accomplish this goal?